### DISCUSSION TITLE
Ripgrep seems to return WAY LESS results than GNU grep

### DISCUSSION THREAD

**Discussion by RVFET** [BurntSushi/ripgrep#2863] [Q&A]
Workdir:
```
.
├── MEGAFILE01.txt
├── MEGAFILE02.txt
├── MEGAFILE03.txt
├── MEGAFILE04.txt
├── MEGAFILE05.txt
├── MEGAFILE06.txt
├── MEGAFILE07.txt
├── MEGAFILE08.txt
├── MEGAFILE09.txt
├── MEGAFILE10.txt
└── MEGAFILE11.txt
```

ripgrep command:
```zsh
rg -oNI ".*SEARCHREGEX.*"
```
Result count: **29**

grep command:
```zsh
grep -r -s -i -a -E --include "*" "SEARCHREGEX"
```
Result count: **85**

Due to sensitivity of the data, I'm unable to show neither the original search regex, nor the results returned. For both commands SEARCHREGEX is a "domain.tld" (eg: google.com).

**Comment by BurntSushi** [comment]
Well without an MRE I can't give you a concrete answer. But I mean, your commands aren't even close to the same, so I'm not sure why you would expect them to produce the same result.
